pkg/webui/console/store/selectors/applications.js

```diff
@@ -90,3 +90,9 @@ export const selectApplicationLinkFormatters = state => {
 
   return link.default_formatters
 }
+
+export const selectApplicationLinkSkipPayloadCrypto = state => {
+  const link = selectApplicationLink(state) || {}
+
+  return link.skip_payload_crypto
```

Comment:
```suggestion
  return link.skip_payload_crypto || false
```

Even though I'm not in the matter enough to determine whether this is a desirable setting. 

@htdvisser if there is no link or the `skip_payload_crypto` prop missing in the AS response, can we assume that payload crypto is not skipped then?

Reply:
Also, how can the user configure `skip_payload_crypto` on the application link level?